docs/internals/howto-release-django.txt

```diff
@@ -364,48 +429,63 @@ OK, this is the fun part, where we actually push out a release!
    ``Django-<version>.checksum.txt.asc`` which you can then verify using ``gpg
    --verify Django-<version>.checksum.txt.asc``.
 
-If you're issuing multiple releases, repeat these steps for each release.
```

Comment:
These short comments after a long enumeration are easily missed, so I moved it to the introductory section paragraph.